user/deployment/snaps.md

```diff
@@ -1,38 +1,69 @@
 ---
-title: Ubuntu Snap Store
+title: Snap Store
 layout: en
 permalink: /user/deployment/snaps/
 ---
 
-Travis CI can automate the continuous delivery to the [Snaps](https://snapcraft.io) Store.
+Travis CI can automatically upload and release your built snap to the [snap store](https://snapcraft.io) after a successful build.
 
 Snaps are a new packaging and delivery system for Ubuntu, Debian, Fedora, openSuse and other [Linux distros](https://snapcraft.io/docs/core/install). Snaps are secure, they bundle all their dependencies and they are designed to be part of the upstream development workflow in a seamless way. This means that no distro maintainers are involved, and that the delivery to the store can be fully automated. The snaps installed in the users' machines are auto-updated, so a few hours after you push a new snap to the store, all your users will get the most recent version.
 
-To automate continuous delivery of snaps to the Ubuntu store:
+All it takes is the following in your `.travis.yml`:
 
-1. [Encrypt](https://docs.travis-ci.com/user/encrypting-files) your snapcraft credentials with Travis CI
-2. Add the encrypted credentials to the repository where you have the `snapcraft.yaml` metadata for your package.
-3.  Run `snapcraft` and `snapcraft push` in your `.travis.yml`:
+```yaml
+deploy:
+  provider: snap
+  snap: my.snap
+  channel: edge
+```
+{: data-file=".travis.yml"}
+
+In order for Travis CI to have permission to upload and release the snap on your behalf, you need to export a login token by installing Snapraft:
+
+```bash
+sudo snap install snapcraft --classic
+```
+
+Now export a login covering the ability to push this specific snap, as well as release to the channel you selected (in this example, `edge`):
+
+```bash
+snap export-login --snaps my-snap-name --channels edge -
+```
 
-    ```yaml
-    deploy:
-      skip_cleanup: true
-      provider: script
-      script: docker run -v $(pwd):$(pwd) -w $(pwd) snapcore/snapcraft sh -c "apt update && snapcraft && snapcraft push *.snap --release edge"
-      on:
-        branch: master
-    ```
+(note the final `-`, which requests the login be exported to stdout instead of a file.) Copy the token, and put it into the `$SNAP_TOKEN` environment variable in Travis CI:
 
-The `snapcraft enable-ci travis` command will assist you getting the credential, encrypting it and adding the right script to the deploy section of your `.travis.yml` file. Here is a [tutorial that will guide you setting up the continuous delivery from Travis CI](https://tutorials.ubuntu.com/tutorial/continuous-snap-delivery-from-travis-ci#0) on your project.
+```bash
+travis env set SNAP_TOKEN "<token>"
+```
+
+If you use a different environment variable, you'll need to specify the `token` option for the provider:
```

Comment:
I think we should drop this. Let's keep them on rails.

Reply:
Sure, although I'm not sure how they'll learn it's an option this way. Done.